backend/src/packages/projects/project.repository.ts

```diff
@@ -42,6 +42,9 @@ class ProjectRepository
           void builder.where('name', 'ilike', `%${name}%`);
         }
       })
+      .orderBy('name')
+      .offset((page - 1) * limit)
+      .limit(limit)
```

Comment:
There is a specific [stuff](https://vincit.github.io/objection.js/recipes/paging.html) for pagination in objection

Reply:
It will return `results` array and `total` counter so you won't need to implement it by yourself